src/newsApi.test.js

```diff
@@ -0,0 +1,18 @@
+const newsApi = require("./newsApi");
+
+require("jest-fetch-mock").enableMocks();
+
+describe(newsApi, () => {
+  it("calls fetch and loads data", () => {
+    const api = new newsApi();
+
+    fetch.mockResponseOnce(
+      JSON.stringify({
+        response: { results: "Some value" },
```

Comment:
I would recommend making this structured as close to the real api data as possible